README.md

```diff
@@ -9,41 +9,67 @@ validating international phone numbers. The Java version is optimized for
-  * ` isNumberMatch ` - gets a confidence level on whether two numbers could be the same.
-  * ` getExampleNumber `/` getExampleNumberByType ` - provides valid example numbers for all countries/regions, with the option of specifying which type of example phone number is needed.
-  * ` isPossibleNumber ` - quickly guessing whether a number is a possible phonenumber by using only the length information, much faster than a full validation.
-  * ` isValidNumber ` - full validation of a phone number for a region using length and prefix information.
-  * ` AsYouTypeFormatter ` - formats phone numbers on-the-fly when users enter each digit.
-  * ` findNumbers ` - finds numbers in text input.
-  * ` PhoneNumberOfflineGeocoder ` - provides geographical information related to a phone number.
-  * ` PhoneNumberToCarrierMapper ` - provides carrier information related to a phone number.
-  * ` PhoneNumberToTimeZonesMapper ` - provides timezone information related to a phone number.
 
-# Demo (v8.6.0)
-[Java](http://libphonenumber.appspot.com/)
-
-[JavaScript](https://rawgit.com/googlei18n/libphonenumber/master/javascript/i18n/phonenumbers/demo-compiled.html)
-
-# Code
-To include the code in your application, either integrate with Maven or
-download the latest Jars from the Maven repository:
-
-http://repo1.maven.org/maven2/com/googlecode/libphonenumber/libphonenumber/
+*   Parsing / formatting / validating phone numbers for all countries / regions
+    of the world.
+*   `getNumberType` - gets the type of the number based on the number itself;
+    able to distinguish Fixed-line, Mobile, Toll-free, Premium Rate, Shared
+    Cost, VoIP and Personal Numbers  (whenever feasible).
+*   `isNumberMatch` - gets a confidence level on whether two numbers could be
+    the same.
+*   `getExampleNumber` / `getExampleNumberByType` - provides valid example
+    numbers for all countries/regions, with the option of specifying which type
+    of example phone number is needed.
+*   `isPossibleNumber` - quickly guessing whether a number is a possible
+    phone number by using only the length information, much faster than a full
+    validation.
+*   `isValidNumber` - full validation of a phone number for a region using
+    length and prefix information.
+*   `AsYouTypeFormatter` - formats phone numbers on-the-fly when users enter
+    each digit.
+*   `findNumbers` - finds numbers in text input.
+*   `PhoneNumberOfflineGeocoder` - provides geographical information related to
+    a phone number.
+*   `PhoneNumberToCarrierMapper` - provides carrier information related to a
+    phone number.
+*   `PhoneNumberToTimeZonesMapper` - provides timezone information related to a
+    phone number.
+
+# Demo
+
+The [Java demo](http://libphonenumber.appspot.com/) is updated periodically,
```

Comment:
Changed from:

```
The [Java demo](http://libphonenumber.appspot.com/) is updated periodically,
with a slight delay after the GitHub release. See the [latest release's version
number](http://github.com/googlei18n/libphonenumber/releases).
```

to:
```
The [Java demo](http://libphonenumber.appspot.com/) is updated with a slight
delay after the GitHub release.
Last demo update: v8.6.0.
If this number is lower than the [latest release's version
number](http://github.com/googlei18n/libphonenumber/releases), we are between
releases and the demo may be at either version.
```

To answer the other comment, I don't think the number on the demo should come from the GitHub page, because that's incorrect too. Between the GitHub release and the demo update it would say the new number but not be updated yet.

I still don't see why anyone would want the number on the README page when it's incorrect some of the time, and they need to know the caveat about it being off from the release. I think the only potential improvement over not having anything there would be to explicitly say the demo **was** updated, so I put that other language in.